### DISCUSSION TITLE
Does the fastvpinns support different variational forms of loss?

### DISCUSSION THREAD

**Discussion by T899work** [community/community#127714] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've noticed that FastVPINNs only incorporates one variational form of loss. Are there more options available in your implementation? I am encountering some obstacles while trying to modify the variational form. I would greatly appreciate any assistance you can provide. (similar to other publicly available VPINN codes)

**Comment by ebndev** [comment]
Hi @T899work, thanks for being a part of the GitHub Community!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from a specific. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

P.S would you mind telling us how you ended up reaching out for support in the Profile Category of Discussions? We are getting a lot of posts where folks are reaching out for support from projects hosted on GitHub and we want to ensure there aren't links or directions out there that may be misguiding folks.

**Comment by thivinanandh** [comment]
hi @T899work , 

Please open a discussion in the fastvpinns repo at https://github.com/cmgcds/fastvpinns/discussions

**Reply by ebndev** [reply]
Hi again @T899work!

If you don't mind, would you mind letting us know how you initially ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your discussion? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board, if that’s where you intended to post.

Also @thivinanandh please feel free to chime in if you found that other users experienced this as well.

Any insight would be helpful, thanks!

**Reply by ebndev** [reply]
Never mind, found the issue! Thanks again for your time and being a part of the GitHub Community 😄
